lib/core/login/components/login.component.ts

```diff
@@ -151,6 +151,13 @@ export class LoginComponent implements OnInit, OnDestroy {
     ) {
         this.initFormError();
         this.initFormFieldsMessages();
+        if (this.authService.isOauth()) {
```

Comment:
I guess will be nice to have some unit test around this area is quite important to have good coverage in the core